inventory/byo/hosts.ose.example

```diff
@@ -271,6 +271,19 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # Setting this variable to true will override that check.
 #openshift_override_hostname_check=true
 
+# Specify which method to use to synchronize system clocks.
+# none   - Do not manage clocks with openshift-ansible.
+# auto   - Automatically choose method based on host type [atomic=chrony, default=ntpd].
```

Comment:
On thinking about this a bit more...  I'm wondering if there should not be an auto option at all, and openshift_clock_method being undefined should default to host detection in openshift_facts...

You might even want to go as far as testing if ntpd or chrony are already installed and use the pre-installed package if present as the default.